CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to PlaceOS are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PlaceOS Platform Versioning](./README.md#platform-versioning).
 
+## Unreleased
+
+- Fix healthcheck of `frontend-loader`.
+- Account for branches in commit listing in `rest-api`.
+- Fix `core` and `frontend-loader` resetting to master on git queries.
+- Fix rethinkdb driver reconnections on writes.
+- Fix token revocation in `auth`.
+- Improve handling of GraphAPI errors in `staff-api`.
```

Comment:
Logging improvements for this were included in `1.2109.0`. Only change of relevance since then appears to be https://github.com/PlaceOS/office365/commit/3a6c1135ad6dacf702ff3398cf08673f281b4200, which is an internal detail.